IBX-2762 Implement measurements conversion (#1991)
ezsystems/developer-documentation: docs/content_management/field_types/field_type_reference/measurementfield.md

!!! note

    To be available for selection in the Back Office, each new Measurement type or unit must be enabled for the  Back Office SiteAccess.

Next, you need to define how the new unit should be converted under the `ibexa.system.<scope>.ibexa_measurement` [configuration key](configuration.md#configuration-files):

```yaml
ibexa_measurement:
    conversion:
        formulas:
            - { source_unit: foo, target_unit: bar, formula: 'value / 100' }
    types:
        length:
            foo: { symbol: foo }
            bar: { symbol: bar }
```

!!! tip

>     The `target_unit` needs to be an existing unit, for example meter, otherwise the conversion will result in an error.

Reviewer: ```suggestion
    The `target_unit` must be an existing unit, for example meter, otherwise the conversion results in an error.
```
